federation/pkg/federation-controller/configmap/configmap_controller.go

```diff
@@ -44,6 +45,11 @@ import (
 
 const (
 	allClustersKey = "ALL_CLUSTERS"
+	ControllerName = "configmaps"
```

Comment:
Does the fact that this matches with the resource type specified on L52 indicate that the indicate that this should be ``ResourceType`` or ``ResourceName``?  Also, consider using this constant in specifying ``RequiredResources``.